tests/pytest/test_deprecated_imports.py

```diff
@@ -0,0 +1,15 @@
+import pytest
+
+
+def test_import_bus():
+    with pytest.warns(DeprecationWarning):
+        import cocotb.bus  # noqa: F401
+
+    with pytest.warns(DeprecationWarning):
+        import cocotb.drivers  # noqa: F401
+
+    with pytest.warns(DeprecationWarning):
+        import cocotb.monitors  # noqa: F401
+
+    with pytest.warns(DeprecationWarning):
+        import cocotb.scoreboard  # noqa: F401
```

Comment:
Add tests for all modules mocked out as well to increase coverage. Maybe use `importlib.import_module` and make this a list of names instead for flexibility and conciseness.

Reply:
Nvm, `import_module` is swallowing the `DeprecationWarning`s for some reason. And the warnings are only printed once.